File: Entities/Models/ProjectModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Application.EntitiesModels.Models
{
    public class ProjectModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public DateTime? DateBegin { get; set; }

        public DateTime? DateEnd { get; set; }

        public int EmploeeCount { get; set; }

        public int? ClientId { get; set; }

        public string OrganizationName { get; set; }

        public IEnumerable<UserProfileProjectModel> Emploees { get; set; }

        public IEnumerable<string> EmploeesAvatars { get; set; }

        public IEnumerable<string> EmploeesPositions { get; set; }

        public string Comment { get; set; }

        public ClientModel Client { get; set; }
    }
}

File: Application/ClientApp/app/models/project.model.ts
import { Client } from "./client.model";
import { User } from "./user.model";
import { UserProfileProject } from "./user-profile-project.model";

export class Project {
    Id: number

    Name: string

    DateBegin: Date;

    DateEnd: Date;

    EmploeeCount: number;

    ClientId: number;

    OrganizationName: string;

    Emploees: Array<UserProfileProject>;

    EmploeesAvatars: Array<string>;

    EmploeesPositions:  Array<string>;

    Comment: string;

    Client: Client;
}